source/index.html.md

```diff
@@ -423,6 +425,16 @@ curl -u :<eclair_api_password> -X POST "http://localhost:8080/channels"
 eclair-cli channels
 ```
 
+The units of returned fields that are not obvious from their names:
+
+field          | unit
+---------------|-----
+dustLimit      | sats
+channelReserve | sats
+htlcMinimum    | ???
+toSelfDelay    | blocks
+feeratePerKw   | sats
```

Comment:
This is `sat/kw` to be more precise (bitcoin feerate units can be very confusing - but it's getting better). This also applies to line 653.

Reply:
Yeah, it's in name so I feared it'd make it look like sat/kw^2 but maybe it's too strange for anyone to seriously think about it?

Reply:
Right, that makes sense, let's keep it like that.